models/task.js

```diff
@@ -37,6 +37,11 @@ module.exports = (sequelize, DataTypes) => {
         Task.hasMany(models.Assign, { foreignKey: 'TaskId' })
         Task.hasMany(models.Offer, { foreignKey: 'taskId' })
         Task.hasMany(models.Member, { foreignKey: 'taskId' })
+        Task.belongsToMany(models.Label, { foreignKey: 'taskId',
+          otherKey: 'labelId',
+          through: 'TaskLabels',
+          onUpdate: 'CASCADE',
+          onDelete: 'CASCADE' })
```

Comment:
@alexanmtz a remove on the label will not remove the task it would only remove the association between such label a task in the join table NOT the label in the label table same goes for the task.

So once you remove task with taskId all the labelId of such task will be removed in the TaskLabels table.

Here is a post which I think might help
https://stackoverflow.com/questions/56213758/on-delete-cascade-for-multiple-foreign-keys-with-sequelize

Kindly let me know if I didn't address the issue.

Reply:
Great @Adekoreday